### ISSUE TITLE
Faire un bandeau par persona (définir, illustrer)

### ISSUE THREAD

**Issue by GuillaumeAV** [chemins-de-la-transition/semapps#470]
Proposition de personas : 

- Particuliers 
- Etudiants, personnes en insertion ou en reconversion
- Entreprises, collectivités 
- Acteurs de la Transition (souhaitant rejoindre les Chemins)

**Comment by clemenceCDLT** [comment]
Yes ! Par porte d'entrée tu entends une redirection vers quelque chose de particulier ?

**Comment by GuillaumeAV** [comment]
Y a du design avant, amis l'idée serait d'arriver sur une page de résultats
classique avec le filtre public cible activé ...

Le ven. 8 juil. 2022 à 08:46, clemenceCDLT ***@***.***> a
écrit :
